### ISSUE TITLE
A textured examples would be useful

### ISSUE THREAD

**Issue by nicolasgramlich** [Unity-Technologies/Unity.Animation.Samples#2]
Currently the examples are lit without a textures. An example with textures would be useful.

**Comment by oldio** [comment]
Agreed, we can definitely add a shader graph with textures next iteration. Otherwise you could always modify or create your own graphs following tutorials:
https://www.youtube.com/watch?v=V5XFrIhLpGQ

**Comment by nicolasgramlich** [comment]
@oldio thanks. Any idea when that next iteration could come around? We're not going to quote you on that ofc, just for our estimations if we should wait or jump into the cold water by ourselves.

**Comment by johnsoncodehk** [comment]
This is a modify shader graph base on Basic_Vertex_Skinning, added main texture and normal map properties: [Basic_Vertex_Skinning_Modify.shadergraph.zip](https://github.com/Unity-Technologies/Unity.Animation.Samples/files/4400073/Basic_Vertex_Skinning_Modify.shadergraph.zip)
